gradio/components/video.py

```diff
@@ -247,7 +247,7 @@ def postprocess(
         Returns:
             VideoData object containing the video and subtitle files.
         """
-        if value is None or value == [None, None] or value == (None, None):
+        if value is None or all(x is None for x in value):
```

Comment:
this is fine, but nit: we should typecheck that `x` is a list/tuple before iterating over it like this